lib/get-config.js

```diff
@@ -366,11 +426,19 @@ function getProjectConfig(options) {
 
     validateRules(config.rules, config.loadedRules, options);
     validateOverrides(config, options);
+    config = processRules(config);
 
     config._processed = true;
   }
 
   return config;
 }
 
-module.exports = { getProjectConfig, getConfigForFile };
+module.exports = {
+  getProjectConfig,
+  getConfigForFile,
+  ERROR_SEVERITY,
+  WARNING_SEVERITY,
+  IGNORE_SEVERITY,
+  processRules,
+};
```

Comment:
Mind if we sort these like this?

```suggestion
module.exports = {
  ERROR_SEVERITY,
  IGNORE_SEVERITY,
  WARNING_SEVERITY,
  getConfigForFile,
  getProjectConfig,
  processRules,
};
```